docs/src/main/paradox/scala/http/client-side/client-transport.md

```diff
@@ -1,9 +1,14 @@
 # Pluggable Client Transports / HTTPS Proxy Support
 
 The client side infrastructure has (unstable) support to plug different transport mechanisms underneath. A client side
-transport is represented by an instance of @scaladoc[akka.http.scaladsl.ClientTransport](akka.http.scaladsl.ClientTransport):
+transport is represented by an instance of
+@scala[@scaladoc[akka.http.scaladsl.ClientTransport](akka.http.scaladsl.ClientTransport)]@java[@javadoc[akka.http.javadsl.ClientTransport](akka.http.javadsl.ClientTransport)]:
 
-@@snip [ClientTransport.scala](../../../../../../../akka-http-core/src/main/scala/akka/http/scaladsl/ClientTransport.scala) { #client-transport-definition }
+Scala
+:  @@snip [ClientTransport.scala](../../../../../../../akka-http-core/src/main/scala/akka/http/scaladsl/ClientTransport.scala) { #client-transport-definition }
+
+Java
+:  @@snip [ClientTransport.scala](../../../../../../../akka-http-core/src/main/scala/akka/http/javadsl/ClientTransport.scala) { #client-transport-definition }
```

Comment:
Here I don't think quoting the API is necessary since the Scaladoc and Javadoc links provides a way to dig deeper.